### ISSUE TITLE
The quality of loop closure is worse than lego_loam

### ISSUE THREAD

**Issue by ViryaGit** [facontidavide/LeGO-LOAM-BOR#3]
when i set loopClosureEnableFlag = true ,then tested on kitti dataset. loop closure donot found, while the lego_loam work well with the same param.

**Comment by facontidavide** [comment]
To make it easier for me to fix this, can you please share the rosbag you are using?

I am working on this branch 

https://github.com/facontidavide/LeGO-LOAM-BOR/pull/new/loop_closure

**Comment by ViryaGit** [comment]
> 
> 
> To make it easier for me to fix this, can you please share the rosbag you are using?
> 
> I am working on this branch
> 
> https://github.com/facontidavide/LeGO-LOAM-BOR/pull/new/loop_closure

tested on this bag file :[http://robotics.ethz.ch/~asl-datasets/segmap/segmap_data/kitti/](url) 

2011_09_30_drive_18.bag |  
-- | --

**Comment by facontidavide** [comment]
Hi, I noticed that you closed the issue. Have you tested the branch I mentioned? Does it work?

**Comment by saimanoj18** [comment]
Please comment if the loop closure issue is fixed.

**Comment by bugerry87** [comment]
I am still unsatisfied about LeGO-LOAM-BOR's performance.
- In the original version I was able to configure working parameters for the VLP-32c, although it is said to be not supported. BOR appears to me to have issues with the ground-detection.
- Furthermore, I somehow believe that the RVIZ camera affects the result oO. As soon I switch to orbit mode, the track runs nowhere.
- Another issue I have with the PCL leaf-size. I enlarged them by 2 for a hot fix.

PS: I'm not just here to complain. I hope I can contribute in the near future.
